pandas/tests/test_downstream.py

```diff
@@ -274,3 +274,23 @@ def __radd__(self, other):
 
     assert right.__add__(left) is NotImplemented
     assert right + left is left
+
+
+@td.skip_if_no("pydantic")
+@td.skip_if_no("pydantic_core")
+@pytest.mark.parametrize("series", [
+    Series([1, 2, 3]),
+    Series(np.array([1, 2, 3])),
```

Comment:
these two are the same

Reply:
yeah, I actually should have used `[1, 2,3]` and `np.array[1, 2, 3]` without the `Series`. I'll fix it.